Makefile.defs

```diff
@@ -102,9 +102,9 @@ endif
 endif
 
 # renovate: datasource=docker depName=golangci/golangci-lint
-GOLANGCILINT_WANT_VERSION = v2.0.0
```

Comment:
IMO we shouldn't update the version with this PR (and let renovate handle the version upgrades)

upstream commit only changed the flag

Reply:
you mean the whole commit (GOLANGCILINT_WANT_VERSION + GOLANGCILINT_IMAGE_SHA + GOLANGCILINT_VERSION) or only GOLANGCILINT_WANT_VERSION?

Reply:
only keep the changes to `GOLANGCILINT_VERSION` (as done in the upstream commit: https://github.com/cilium/cilium/pull/38996/commits/bcd193f6537609405ddc3af002eeaf0e18048ae2)

Reply:
correct, thank you, fixed.